### ISSUE TITLE
change of term basisOfRecord may impact OBIS

### ISSUE THREAD

**Issue by ymgan** [iobis/quality-taskteam#11]
Yi-Ming Gan
  [9:47 AM](https://obishq.slack.com/archives/C013QC3AYH0/p1662968865414719)
There is a proposal to change the term basisOfRecord. I think it might impact OBIS because:
Proposed attributes of the new term version (Please put actual changes to be implemented in bold and strikethrough):
Examples (not normative): Preserved Specimen, Fossil Specimen, Living Specimen, Material Sample, Event, Human Observation, Machine Observation, Taxon, Occurrence, Material Citation
GitHub issue: https://github.com/tdwg/dwc/issues/416

**Comment by ymgan** [comment]
Does OBIS require basisOfRecord only because it's required by the IPT? Or would we still require it even if it was no longer required in the IPT? 



We inherited the requirement from https://rs.gbif.org/core/dwc_occurrence_2022-02-02.xml, but I think it's a very useful field to have and would suggest making it (at least) recommended.



From the ecological perspective, I think it is a very useful field.




Thanks Pieter that's what I thought. I have mixed feelings about the field. There have been many very long round and round discussions about it (e.g. [this one](https://github.com/tdwg/dwc/issues/302)) and what it means. Aggregators want to use it to be able to filter data but that's not what it was intended for so this use of that field for that purpose has created tension. I think we are all piling a bit too much onto this one field and have too many differing expectations of it.
